src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -276,6 +276,10 @@ public static String getShowDeletedValue(String showDeleted) {
         return "True".equalsIgnoreCase(showDeleted) ? ProfessionalApiConstants.TRUE : ProfessionalApiConstants.FALSE;
     }
 
+    public static String getReturnRolesValue(String returnRoles) {
+        return "False".equalsIgnoreCase(returnRoles) ? ProfessionalApiConstants.FALSE : ProfessionalApiConstants.TRUE;

```

Comment:
can you create "False" and "True" as ProfessionalApiConstants ? Also replace wherever required I class.

Reply:
They have been used already, in the same way the `showDeleted` method is done above

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/603/commits/52c2dde783fadd143910ebd830745d2f095df9e9